tests/integration/ipfs.test.js

```diff
@@ -0,0 +1,20 @@
+import { writeToIPFS, dereferenceFromIPFS } from '../../src/utils/rdf';
+
+const rdfInput = `
+  @prefix : <http://example.org/stuff/1.0/> .
+  :a :b ( "apple" "banana" ) .
+`;
+
+const rdfCID = 'QmQeskBk9TsdWXaNX26PvuzkxU8zzoWZo5dmJMPuw5vcSS';
+
+describe('IPFS', () => {
+  test('Can write document to IPFs', async () => {
+    const cid = await writeToIPFS(rdfInput);
+    expect(cid.toString()).toEqual(rdfCID);
+  }, 10000);
+
+  test('Can dereference document from IPFs', async () => {
+    const document = await dereferenceFromIPFS(rdfCID);
+    expect(document).toEqual(rdfInput);
+  }, 10000);
+});
```

Comment:
Is it possible to add a test to prove that dereferencing a live *ipns* document results in an error?

Reply:
why would dereferencing a live ipns document result in an error?

Reply:
Ipns docs are mutable. This prevents a verifier from knowing when to invalidate data that was pulled derived from that sort of document.

Let's say the crawler extracts a claimgraph from an ipns document. There isn't a straightforward way for the verifier to know that the document changed and that a re-crawl is required.